scripts/linters/test_files/invalid_unknown_in_ts_file.spec.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview A service for generating random and unique content_id for
+ * SubtitledHtml domain objects.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
```

Comment:
These test cases seem overly complicated when we really just need to test combinations of comments and `unknown` types. I'd recommend providing these test cases as strings instead so you can easily write more of them (and catch some of the edge cases I flagged)